python/cairo-core/src/cairo_core/maths.cairo

```diff
@@ -536,3 +536,61 @@ func felt252_bit_length{range_check_ptr}(value: felt) -> felt {
 
     return bit_length;
 }
+
+// @notice Converts a felt252 to a bit array, little-endian, and outputs to `dst`.
+// @dev Can only convert up to 251 bits included.
+// @returns the length of the bit array.
+func felt252_to_bits_rev{range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(
```

Comment:
why do we call it rev instead of felt252_to_le_bits?

Reply:
endianness is the order in which bytes within a word of digital data are stored, thus i don't think it applies to bits